### PR TITLE
fix: fix presubmit

### PR DESCRIPTION
**Pull request by wcalandro**
Make sure our dependencies are installed in BazelCI

### CODE REVIEW

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 tasks:
   ubuntu_2004:
     platform: ubuntu2004
+    shell_commands:
+      - apt-get -y install bison flex uuid-dev asciidoc graphviz source-highlight
```
[the remark this answers is not in the document]
Comment (wcalandro) [reply .bazelci/presubmit.yml]: sudo: account validation failure, is your account locked?

[the remark this answers is not in the document]
Comment (shahms) [reply .bazelci/presubmit.yml]: Oh, have to use `apt` on ubuntu 20.04, not `apt-get`:

(From https://github.com/bazelbuild/continuous-integration/pull/1236/files#diff-4b8557b9f4d31430f4624802c2b13d81c4334879151dafaa5fdcd941e5f7932b)

**Comment by shahms** [comment]
1804 has an older version of Clang which doesn't work for us.

File: .bazelci/presubmit.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (philwo) [reply .bazelci/presubmit.yml]: I found the issue why sudo was not working in the Ubuntu 20.04 container and fixed it: https://github.com/bazelbuild/continuous-integration/commit/3dc99b38b749686cd623f24da800fc326569d5d4

Can you please retry with the ubuntu2004 platform? It should work now. I'm sorry for the issues.

**Pull request merged**